packages/kbn-text-based-editor/src/errors_warnings_popover.tsx

```diff
@@ -0,0 +1,203 @@
+        values: { count },
+      }),
+      label: i18n.translate('textBasedEditor.query.textBasedLanguagesEditor.warningsTitle', {
+        defaultMessage: 'Warnings',
+      }),
+    };
+  }
+};
+
+export function ErrorsWarningsContent({
+  items,
+  type,
+  onErrorClick,
+}: {
+  items: MonacoMessage[];
+  type: 'error' | 'warning';
+  onErrorClick: (error: MonacoMessage) => void;
+}) {
+  const strings = getConstsByType(type, items.length);
+  return (
+    <div style={{ width: 500 }}>
+      <EuiPopoverTitle paddingSize="s">{strings.label}</EuiPopoverTitle>
+      <EuiDescriptionList>
+        {items.map((item, index) => {
+          return (
+            <EuiDescriptionListDescription
+              key={index}
+              className={classNameCss`
+                                &:hover {
+                                  cursor: pointer;
+                                }
+                              `}
+              onClick={() => onErrorClick(item)}
+            >
+              <EuiFlexGroup gutterSize="xl" alignItems="flexStart">
+                <EuiFlexItem grow={false}>
+                  <EuiFlexGroup gutterSize="s" alignItems="center">
+                    <EuiFlexItem grow={false}>
+                      <EuiIcon type={type} color={strings.color} size="s" />
+                    </EuiFlexItem>
+                    <EuiFlexItem style={{ whiteSpace: 'nowrap' }}>
+                      {i18n.translate('textBasedEditor.query.textBasedLanguagesEditor.lineNumber', {
+                        defaultMessage: 'Line {lineNumber}',
+                        values: { lineNumber: item.startLineNumber },
+                      })}
+                    </EuiFlexItem>
+                  </EuiFlexGroup>
+                </EuiFlexItem>
+                <EuiFlexItem grow={false} className="TextBasedLangEditor_errorMessage">
+                  {item.message}
+                </EuiFlexItem>
+              </EuiFlexGroup>
+            </EuiDescriptionListDescription>
+          );
+        })}
+      </EuiDescriptionList>
+    </div>
+  );
+}
+
+export function ErrorsWarningsCompactViewPopover({
+  items,
+  type,
+  onErrorClick,
+  popoverCSS,
+}: {
+  items: MonacoMessage[];
+  type: 'error' | 'warning';
+  onErrorClick: (error: MonacoMessage) => void;
+  popoverCSS: Interpolation<Theme>;
+}) {
+  const [isPopoverOpen, setIsPopoverOpen] = useState(false);
+  const strings = getConstsByType(type, items.length);
+  return (
+    <EuiPopover
+      button={
+        <EuiBadge
+          color={strings.color}
+          onClick={() => setIsPopoverOpen(true)}
+          onClickAriaLabel={strings.message}
+          iconType={type}
+          iconSide="left"
+          data-test-subj={`TextBasedLangEditor-inline-${type}-badge`}
+          title={strings.message}
+          css={css`
+            cursor: pointer;
+          `}
+        >
+          {items.length}
+        </EuiBadge>
+      }
+      css={popoverCSS}
```

Comment:
I'm not sure this is correct: the previous CSS style was applied to the EUIBadge, but now it is passed and applied to the Popover.
I can see for example that the mouse cursor is not working very well there,it should show always the pointer but instead it just shows the pointer over the warning counter.

https://github.com/elastic/kibana/assets/1421091/e650218f-f065-41c0-a5fc-c34f728aeba7

Reply:
@markov00 I can't replicate that or I am misunderstanding your comment. It is a pointer wherever I hover even if the popover is open or not. Which browser are you using?

Reply:
Ah sorry I believe this was happening right before Marco commented and your fix in [b1bdfa7](https://github.com/elastic/kibana/pull/176568/commits/b1bdfa79cfa2eaeaf42787479df6f57f950f7878) I believe we where reviewing together at the same time but I was late :D